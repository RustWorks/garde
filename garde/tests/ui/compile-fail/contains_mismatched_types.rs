static NUM: u32 = 666;

#[derive(garde::Validate)]
struct Test {
    #[garde(contains(NUM))]
    field: String,
}

fn main() {}
