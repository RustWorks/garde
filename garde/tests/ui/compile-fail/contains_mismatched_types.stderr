error[E0308]: mismatched types
 --> tests/ui/compile-fail/contains_mismatched_types.rs:5:22
  |
5 |     #[garde(contains(NUM))]
  |                      ^^^ expected `&str`, found `&u32`
  |
  = note: expected reference `&str`
             found reference `&u32`
